homeworks/ramiro-miguez/01-html/contact.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Contact Me</title>
+  </head>
+  <body>
+    <nav>
+      <a href="index.html">Home</a> |
+      <a href="contact.html">Contact</a>
+    </nav>
+    <section>
+      <h1>Contact Me</h1>
+      <form action="" method="post">
+        <ul>
+          <li>
+            <label for="fname">Full name*</label>
+            <input
+              type="text"
+              id="fname"
+              name="firstname"
+              placeholder="Your name.."
+            />
+          </li>
+          <li>
+            <label for="fname">Full name*</label>
+            <input type="text" name="lastname" placeholder="Your last name.." />
+          </li>
+          <li>
+            <label for="phone">Phone number*</label>
+            <input
+              type="tel"
+              id="phone"
+              name="phone"
+              placeholder="Your phone.."
+            />
+          </li>
+          <li>
+            <label for="country">Country</label>
+            <select id="country" name="country">
+              <option value="australia">Australia</option>
+              <option value="argentina">Argentina</option>
+            </select>
+          </li>
+          <li>
+            <label for="message"></label>
+            <textarea
+              id="message"
+              name="message"
+              placeholder="Your message.."
+              style="height: 200px"
+            ></textarea>
+          </li>
+          <li><input type="submit" value="Submit" /></li>
+        </ul>
+      </form>
+    </section>
+    <footer>
+      <a href="https://www.facebook.com">Facebook</a>
+      <a href="https://www.instagram.com">Instagram</a>
+      <a href="https://www.linkedin.com">LinkedIn</a>
+      <a href="https://www.twitter.com">Twitter</a>
+    </footer>
```

Comment:
<img width="315" alt="image" src="https://github.com/JulianOviedo/SOLVD-FE-training/assets/97484290/0434a9b7-4723-4ed9-a45a-0ab36482d04e">
doesnt looks like the example
also, i think the dots on text area and the button looks bad